ansible/roles/jenkins/tasks/tools.yml

```
@@ -0,0 +1,40 @@
+---
+
+- name: Upload gcp credentials
+  ansible.builtin.copy:
+    src: "whanos_gcp_creds.json"
+    dest: /var/lib/jenkins/whanos_gcp_creds.json
+    mode: preserve
+  become: true
+
+- name: Zip docker images folder
+  ansible.builtin.archive:
+    path: "{{ playbook_dir }}/jenkins/files/docker_images"
+    dest: docker_images.zip
+    format: zip
+  delegate_to: localhost
+  become: false
+
+- name: Zip scripts folder
+  ansible.builtin.archive:
```

fqcn[canonical]: You should use canonical module name `community.general.archive` instead of `ansible.builtin.archive`
```suggestion
  community.general.archive:
```